src/test/java/seedu/address/logic/parser/KeywordParserTest.java

```diff
@@ -15,24 +14,73 @@
 
 
 public class KeywordParserTest {
+    @Test
+    public void parseInputWithName() {
+        String[] input = {"John", "Doe"};
+        Predicate<Person> predicate = KeywordParser.parseInput(input);
+        assertTrue(predicate instanceof NameContainsKeywordsPredicate);
+    }
+
+    @Test
+    public void parseInputWithIC() {
```

Comment:
I think for these tests can try follow the convention where the name of the method should have the expected outcome and stuff like that.

Reply:
Yes will make the edits